app/views/partial_availabilities/new.html.haml

```diff
@@ -0,0 +1,22 @@
+= content_for :h1 do
```

Comment:
It feels a little weird that it loses the tabs once you click into this view, but that's also what we do for offline, so I guess I'm ok with it.

Reply:
Yeah, I think there are improvements to be made here, but I was just mimicking the established pattern in this area.